web/src/components/home/subcomponents/welcome/welcome.styles.js

```diff
@@ -0,0 +1,42 @@
+import theme from "../../../../common/theme";
+
+const welcomeStyles = {
+  sectionLayout: {
+    display: "flex",
+    flexDirection: { xs: "column", md: "row" },
+    justifyContent: "space-between",
+    maxHeight: { xs: "40rem", sm: "50rem", md: "35rem" },
+  },
+  splashPicture: {
+    objectFit: "cover",
+    height: { xs: "20rem", sm: "25rem", md: "35rem" },
+    width: "auto",
+    aspectRatio: "3/2",
+    overflow: "hidden",
+  },
+  titleBox: {
+    padding: "8vw",
+    whiteSpace: "nowrap",
+  },
+  title: {
+    color: theme.palette.text.title,
+    fontSize: { xs: "2.5rem", md: "4rem" },
+    marginBottom: "1rem",
+  },
+  subtitle: {
+    color: theme.palette.text.secondary,
+    fontSize: { xs: "1.5rem", md: "2.5rem" },
+    marginBottom: "-.8rem",
+  },
+  startHereButton: {
+    backgroundColor: theme.palette.text.textButton,
```

Comment:
BackgroundColor isn't actually text color. Should we create a new bucket instead of adding to this to the text part?

Reply:
actually I can just use the darkPurple background color here